mantle/cmd/kola/kola.go

```diff
@@ -469,17 +467,11 @@ func runIgnitionConvert2(cmd *cobra.Command, args []string) error {
 	return err
 }
 
-func runArtifactIgnitionVersion(cmd *cobra.Command, args []string) error {
```

Comment:
Why was this function removed?  Was it not used anywhere?

Reply:
The beauty of GoLang is that it won't compile if it was being used. However, @bh7cw, mind putting the specific `golangci-lint` errors that your fixing would be excellent.

Reply:
> The beauty of GoLang is that it won't compile if it was being used. However, @bh7cw, mind putting the specific `golangci-lint` errors that your fixing would be excellent.

Sounds good. For this PR, I am fixing:
```
cmd/kola/kola.go:118:2: `extDependencyDir` is unused (deadcode)
        extDependencyDir string
        ^
cmd/kola/kola.go:472:6: `runArtifactIgnitionVersion` is unused (deadcode)
func runArtifactIgnitionVersion(cmd *cobra.Command, args []string) error {
     ^
cmd/kola/kola.go:482:13: Error return value of `errors.New` is not checked (errcheck)
                errors.New("Error: missing required argument --build")
                          ^
```
> Why was this function removed? Was it not used anywhere?

Yes, it's not used now. According to the history, calling the method is removed in https://github.com/coreos/coreos-assembler/commit/419efd209b91b756ec9f801dced7a91f654591eb#diff-305d34ffd6d7cfaa04f3500c473a70ef59613b83e0bdfbb2a44e437774b2886e